declarations/Honkai Star Rail.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "Honkai Star Rail",
```

```suggestion
  "name": "Honkai: Star Rail",
```

We can also maintain the original name of the service as our `service name`